app/views/application/_mailer_doctype.html.haml

```diff
@@ -1 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
I want to note that we're moving from 4.0 transitional to an html 5 header here. The little research I did said 'Stick with the lower header unless you plan on using HTML5 elements. Also it doesn't matter that much because most email clients ignore it and do whatever they want.' It was also from a few years ago, so I imagine HTML is more the expected standard by now. I'm guessing it's fine.

http://blog.mailermailer.com/email-design/html5-doctype-in-email-too-soon
http://stackoverflow.com/questions/34319889/doctype-for-html-email